applications/clock_app/clock_app.c

```diff
@@ -0,0 +1,166 @@
+#include <furi.h>
+#include <furi_hal.h>
+
+#include <gui/elements.h>
+
+#include <gui/gui.h>
+#include <input/input.h>
+
+#define TAG "Clock"
+#define CLOCK_DATE_FORMAT "%.4d-%.2d-%.2d"
+#define CLOCK_TIME_FORMAT "%.2d:%.2d:%.2d"
+
+bool timerStarted=false;
+int timerSecs=0;
```

Comment:
Don't use global variables, make a state structure and allocate it while app is running.